packages/fractor-typoscript/src/ValueObject/TypoScriptPrettyPrinterFormatConfiguration.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\FractorTypoScript\ValueObject;
+
+use a9f\FractorTypoScript\Configuration\TypoScriptProcessorOption;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+
+final readonly class TypoScriptPrettyPrinterFormatConfiguration
+{
+    public function __construct(
+        public int $size,
+        public string $style,
+        public bool $addClosingGlobal,
+        public bool $includeEmptyLineBreaks,
+        public bool $indentConditions
+    ) {
+    }
+
+    public static function createFromParameterBag(ParameterBagInterface $parameterBag): self
+    {
+        $size = $parameterBag->has(TypoScriptProcessorOption::INDENT_SIZE)
+            ? $parameterBag->get(TypoScriptProcessorOption::INDENT_SIZE)
```

Comment:
why not just cast it here as int?

Reply:
Because PhpStan complains that "array" cannot be casted to int/string.